CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ include(RDMA_DoFixup)
 include(publish_headers)
 include(rdma_functions)
 include(rdma_man)
+include(SSEFlags)
```

Comment:
Often we've just been inlining these a little lower in cmakefile, after the compiler is basically setup.

But there is a big grouping of c compiler related stuff now in buildlib/RDMA_EnableCStd.cmake, so you could add a new function there instead of this file..